config/ftbquests/quests/reward_tables/common.snbt

```diff
@@ -0,0 +1,394 @@
+{
+	id: "06C4634E81851A6C"
+	order_index: 12
+	title: "Common Reward"
+	icon: {
+		id: "ftbquests:lootcrate"
+		Count: 1b
+		tag: {
+			type: "common_loot"
+		}
+	}
+	loot_size: 1
+	rewards: [
+		{ item: "botanypots:terracotta_hopper_botany_pot", weight: 5 }
+		{ item: "mysticalagriculture:imperium_essence" }
+		{ item: "reliquary:fertile_lily_pad" }
+		{ item: "minecraft:fox_spawn_egg", weight: 5 }
+		{ item: "functionalstorage:copper_upgrade", weight: 5 }
+		{ item: "functionalstorage:oak_1", random_bonus: 2, weight: 7 }
+		{ item: "functionalstorage:void_upgrade", weight: 3 }
+		{ item: "functionalstorage:storage_controller" }
+		{
+			item: {
+				id: "silentgear:sturdy_repair_kit"
+				Count: 1b
+				tag: {
+					Storage: { }
+				}
+			}
+			weight: 5
+		}
+		{ item: "waystones:waystone", weight: 3 }
+		{
+			item: {
+				id: "utilitix:mob_yoinker"
+				Count: 1b
+				tag: {
+					filled: 0b
+				}
+			}
+			weight: 3
+		}
+		{ item: "waystones:warp_plate", count: 2, weight: 3 }
+		{ item: "dankstorage:dank_1", weight: 5 }
+		{
+			item: {
+				id: "simplemagnets:advancedmagnet"
+				Count: 1b
+				tag: { }
+			}
+			weight: 2
+		}
+		{ item: "cookingforblockheads:sink", weight: 5 }
+		{ item: "ironfurnaces:augment_speed", weight: 5 }
+		{ item: "ironfurnaces:augment_factory", weight: 5 }
+		{ item: "ironfurnaces:item_spooky", weight: 3 }
+		{ item: "ars_nouveau:glyph_summon_wolves" }
+		{ item: "ars_nouveau:glyph_light" }
+		{ item: "pipez:universal_pipe", count: 8, random_bonus: 8, weight: 5 }
+		{ item: "minecraft:diamond", random_bonus: 2, weight: 3 }
+		{
+			item: {
+				id: "simplemagnets:basicmagnet"
+				Count: 1b
+				tag: { }
+			}
+			weight: 4
+		}
+		{ item: "torchmaster:megatorch", weight: 5 }
+		{ item: "productivebees:upgrade_base", weight: 2 }
+		{ item: "sophisticatedstorage:basic_to_iron_tier_upgrade", weight: 5 }
+		{ item: "sophisticatedstorage:upgrade_base", weight: 5 }
+		{ item: "sophisticatedbackpacks:upgrade_base", weight: 5 }
+		{
+			item: {
+				id: "sophisticatedbackpacks:backpack"
+				Count: 1b
+				tag: {
+					inventorySlots: 27
+					upgradeSlots: 1
+					contentsUuid: [I;
+						-104092293
+						790252878
+						-1915449267
+						2101271446
+					]
```

Comment:
Leaving this tag in will cause all backpacks obtained from random quest rewards to be linked together and share the same inventory, as backpacks with the same UUID share the same inventory due to how they're implemented.

(It's almost like a disk from Refined Storage, except these backpacks have a limit of 27-120 item slots instead of unlimited)

Reply:
So essentially, if someone were to get a backpack from one random reward, then fill it up, then get another backpack later down the road, it'd have all of the same stuff duped from the first one?

Reply:
That could happen, but only if the two backpacks were obtained from the same random reward table, which is possible.

Crafting two of the same backpack would give each backpack a different UUID, which is the intended behavior.

(Try removing the NBT data from the reward item and see what happens)

Reply:
After testing this theory, it did not dupe any of the items upon receiving a backpack with the same UUID within the same loot pool.